support/client/lib/vwf/model/mil-sym.js

```diff
@@ -422,6 +443,34 @@ define( [ "module",
                             }
                             break;
 
+                        case "x":
+                        case "y":
+                        case "width":
+                        case "height":
+                        case "rotation":
+                        case "controlPts":
+                        case "symbolType":
+                        case "":
```

Comment:
Is there really an empty-string property name?

Reply:
Over-zealous copy/pasting.  Thanks for catching that.